File: Sum/Sum.py
# prompt user for the size of the list
size = int(input("Enter the size of the array: "))

# initialize list
array = list()

# fill list with "size" number of elements
for i in range(0, size):
    array.append(int(input("Enter element {}: ".format(i))))

# calculate the sum of the elements
sum = 0
for i in array:
    sum = sum + i

# print out the sum
print("The sum of all elements in the array is {}.".format(sum))

File: Sum/Sum.java
import java.util.Scanner;

public class Sum {

    public static void main(String[] args) {
        int size;
        int sum = 0;
        Scanner input = new Scanner(System.in);
        
        // prompt user for size of array
        System.out.print("Enter the number of elements to be stored in the array: ");
        size = input.nextInt();
        
        // initialize array with entered size
        int array[] = new int[size];
        
        // have user fill array
        for(int i = 0;i < size;i++){
            System.out.print("Enter element "+i+": ");
            array[i] = input.nextInt();
        }
        
        // calculate the sum of all of the elements
        for(int i = 0;i < size;i++){
        sum += array[i];
        }
        
        // print out the sum
        System.out.print("The sum of all elements in the array is "+sum+".\n");
    }
    
}
